content/overview/technology/responsibilities.md

```diff
@@ -17,7 +17,7 @@ Here's a chart to illustrate this in three example use cases:
 
 App #1 uses a [Cloud Foundry standard buildpack]({{< relref "docs/getting-started/concepts.md#buildpacks" >}}). (A buildpack provides support for a programming language.) The customer is only responsible for the app code and its dependencies.
 
-App #2 uses an unsupported/custom buildpack, so the customer's responsibility expands from the app code to managing the unsupported buildpack and its dependencies. If you choose to use a custom buildpack, you are responsible for:
+App #2 uses an unsupported/custom buildpack, so the customer's responsibility expands from the app code to managing the custom buildpack and its dependencies. If you choose to use a custom buildpack, you are responsible for:
```

Comment:
`managing the custom buildpack` -> `managing the unsupported/custom buildpack` to be consistent. thoughts?